src/components/victory-axis.jsx

```diff
@@ -47,9 +47,17 @@ const styles = {
   }
 };
 
-class VAxis extends React.Component {
+@Radium
+export default class VictoryAxis extends React.Component {
+  static role = "axis";
```

Comment:
Actually it looks like I just got the syntax wrong and you can't set `prototype` properties using `class` syntax. That's extremely lame, this was a common pattern before ES6 came along. So `instanceof` should work, but I still think it's a weaker solution than using a `role` property (which we can just set in the constructor), or use on the `type` like you've done here. IMO using the exact `displayName` is the most fragile of all options and we'd have to change it sooner rather than later.